spec/lib/sufia/user_stat_importer_spec.rb

```diff
@@ -137,6 +177,7 @@
       context "both error out completely" do
         before do
           expect(FileDownloadStat).to receive(:ga_statistics).exactly(12).times.and_raise(StandardError.new("GA error"))
+          #expect(WorkViewStat).to receive(:ga_statistics).exactly(12).times.and_raise(StandardError.new("GA error"))
```

Comment:
Still sorting out the spec. Those were items I'm still working on.

On Wed, Feb 17, 2016 at 12:31 PM, Justin Coyne <notifications@github.com>
wrote:

> In spec/lib/sufia/user_stat_importer_spec.rb
> <https://github.com/projecthydra/sufia/pull/1564#discussion_r53199877>:
>
> > @@ -137,6 +177,7 @@
> >        context "both error out completely" do
> >          before do
> >            expect(FileDownloadStat).to receive(:ga_statistics).exactly(12).times.and_raise(StandardError.new("GA error"))
> > +          #expect(WorkViewStat).to receive(:ga_statistics).exactly(12).times.and_raise(StandardError.new("GA error"))
>
> Is this needed or not?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/projecthydra/sufia/pull/1564/files#r53199877>.
>